otter/tap/api.py

```diff
@@ -223,6 +226,15 @@ def makeService(config):
         admin_service = service(str(admin_port), admin_site)
         admin_service.setServiceParent(s)
 
+    # setup cloud feed,
+    cf_conf = config['cloudfeeds']
+    cf = CloudFeeds(reactor=reactor, authenticator=authenticator,
+                    region=region, tenant_id=cf_conf['tenant_id'],
+                    service_configs=service_configs)
+    # TODO: It would be nice to not store this as global state but
+    # observer factory is called by twistd before this step
+    set_cloud_feeds(cf)
```

Comment:
We should just do something like: log.addObserver(partial(observe_cloud_feeds, cf)). Granted, the whole log observer system is global, but it shouldn't require us to maintain our own global variables.